packages/web/src/pages/sign-on-page/AudiusValues.tsx

```diff
@@ -51,11 +54,14 @@ type AudiusValuesProps = {
 
 export const AudiusValues = (props: AudiusValuesProps) => {
   const { isDesktop } = useMedia()
+  const tooSmall = useMediaQuery('(max-width: 1363px) and (min-width: 860px)')
```

Comment:
lmk if you have a better name. its the point at which we need to dynamically change the font size so it does overflow

Reply:
maybe `isSmallDesktop`? Seems like we could consider these "small desktop" sizes